src/modules/sensors.cpp

```diff
@@ -18,22 +20,30 @@ static struct pt sensorsThread;
 static struct pt sensorsPressureThread;
 static struct pt sensorsAirFlowThread;
 static struct pt sensorsBatteryThread;
+static struct timer pressureTimer;
+static struct timer airflowTimer;
 
 uint32_t last_debug_time;
 
+#define PRESSURE_SAMPLING_RATE 50000
```

Comment:
Can we move this `#define`s to the top of the file. Also, please use the `MSEC` units so its more clear the units:
```suggestion
#define PRESSURE_SAMPLING_RATE (50 MSEC)
```

Reply:
When I did that, I got an overflow warning.

Reply:
Really? these should be the same since 50 MSEC should be preprocessed to 50 * 1000

Reply:
I know--it's very strange. I figured I'd leave it 'til we got the timer stuff updated.